### PR TITLE
en, zh: add release notes for v1.2.0-rc.2

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add release notes for v1.2.0-rc.2

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/releases/release-1.2.0-rc.2.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## 优化提升
+
+- 支持在未为 BR `toolImage` 指定 tag 时将 TiKV 版本作为 tag ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
+- 支持在扩缩容 TiDB 过程中协调 PVC ([#4033](https://github.com/pingcap/tidb-operator/pull/4033), [@csuzhangxc](https://github.com/csuzhangxc))
+- 为 TiDB Operator 增加 liveness 与 readiness probe ([#4002](https://github.com/pingcap/tidb-operator/pull/4002), [@mikechengwei](https://github.com/mikechengwei))
+
+## Bug 修复
+
+- 修复部署异构集群时 TiDB Operator 可能 panic 的问题 ([#4054](https://github.com/pingcap/tidb-operator/pull/4054) [#3965](https://github.com/pingcap/tidb-operator/pull/3965), [@KanShiori](https://github.com/KanShiori) [@july2993](https://github.com/july2993))
+- 修复 TidbCluster spec 未更改时 TiDB service 与 TidbCluster 状态仍然持续被更新的问题 ([#4008](https://github.com/pingcap/tidb-operator/pull/4008), [@july2993](https://github.com/july2993))
```
Comment (qiancai) [inline zh/releases/release-1.2.0-rc.2.md:27]:
```suggestion
- 修复当 TidbCluster spec 未更改时 TiDB service 与 TidbCluster 状态仍然持续被更新的问题 ([#4008](https://github.com/pingcap/tidb-operator/pull/4008), [@july2993](https://github.com/july2993))
```

File: zh/releases/release-1.2.0-rc.2.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- 支持透传 TiCDC 的 TOML 格式配置 ([#4010](https://github.com/pingcap/tidb-operator/pull/4010), [@july2993](https://github.com/july2993))
+- 支持为 TiCDC 设置 `StorageVolumes`、`AdditionalVolumes` 和 `AdditionalVolumeMounts` ([#4004](https://github.com/pingcap/tidb-operator/pull/4004), [@csuzhangxc](https://github.com/csuzhangxc))
+- 支持为 Discovery、TidbMonitor 和 TidbInitializer 设置自定义 `labels` 与 `annotations` ([#4029](https://github.com/pingcap/tidb-operator/pull/4029), [@csuzhangxc](https://github.com/csuzhangxc))
+- 支持修改 Grafana dashboard ([#4035](https://github.com/pingcap/tidb-operator/pull/4035), [@mikechengwei](https://github.com/mikechengwei))
+
+## 优化提升
+
+- 支持在未为 BR `toolImage` 指定 tag 时将 TiKV 版本作为 tag ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
+- 支持在扩缩容 TiDB 过程中协调 PVC ([#4033](https://github.com/pingcap/tidb-operator/pull/4033), [@csuzhangxc](https://github.com/csuzhangxc))
+- 为 TiDB Operator 增加 liveness 与 readiness probe ([#4002](https://github.com/pingcap/tidb-operator/pull/4002), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (qiancai) [inline zh/releases/release-1.2.0-rc.2.md:22]: probe 中文是什么呢？liveness 与 readiness probe 算两个 probe? 这个对用户有什么好处呢，可以说明下吗？

File: en/releases/release-1.2.0-rc.2.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+## New features
+
+- Support passing raw TOML config for TiCDC ([#4010](https://github.com/pingcap/tidb-operator/pull/4010), [@july2993](https://github.com/july2993))
+- Support setting `StorageVolumes`, `AdditionalVolumes` and `AdditionalVolumeMounts` for TiCDC ([#4004](https://github.com/pingcap/tidb-operator/pull/4004), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support setting custom `labels` and `annotations` for Discovery, TidbMonitor and TidbInitializer ([#4029](https://github.com/pingcap/tidb-operator/pull/4029), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support modifying Grafana dashboard ([#4035](https://github.com/pingcap/tidb-operator/pull/4035), [@mikechengwei](https://github.com/mikechengwei))
+
+## Improvements
+
+- Support using the TiKV version as the tag for BR `toolImage` if no tag is contained ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
```
Comment (qiancai) [inline en/releases/release-1.2.0-rc.2.md:20]:
```suggestion
- Support using the TiKV version as the tag for BR `toolImage` if no tag is specified ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.2.0-rc.2.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# TiDB Operator 1.2.0-rc.2 Release Notes
+
+Release date: July 2, 2021
+
+TiDB Operator version: 1.2.0-rc.2
+
+## New features
+
+- Support passing raw TOML config for TiCDC ([#4010](https://github.com/pingcap/tidb-operator/pull/4010), [@july2993](https://github.com/july2993))
+- Support setting `StorageVolumes`, `AdditionalVolumes` and `AdditionalVolumeMounts` for TiCDC ([#4004](https://github.com/pingcap/tidb-operator/pull/4004), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (qiancai) [inline en/releases/release-1.2.0-rc.2.md:14]:
```suggestion
- Support setting `StorageVolumes`, `AdditionalVolumes`, and `AdditionalVolumeMounts` for TiCDC ([#4004](https://github.com/pingcap/tidb-operator/pull/4004), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.2.0-rc.2.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+- Support setting custom `labels` and `annotations` for Discovery, TidbMonitor and TidbInitializer ([#4029](https://github.com/pingcap/tidb-operator/pull/4029), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (qiancai) [inline en/releases/release-1.2.0-rc.2.md:15]:
```suggestion
- Support setting custom `labels` and `annotations` for Discovery, TidbMonitor, and TidbInitializer ([#4029](https://github.com/pingcap/tidb-operator/pull/4029), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: zh/releases/release-1.2.0-rc.2.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+发布日期：2021 年 7 月 2 日
+
+TiDB Operator 版本：1.2.0-rc.2
+
+## 新功能
+
+- 支持透传 TiCDC 的 TOML 格式配置 ([#4010](https://github.com/pingcap/tidb-operator/pull/4010), [@july2993](https://github.com/july2993))
+- 支持为 TiCDC 设置 `StorageVolumes`、`AdditionalVolumes` 和 `AdditionalVolumeMounts` ([#4004](https://github.com/pingcap/tidb-operator/pull/4004), [@csuzhangxc](https://github.com/csuzhangxc))
+- 支持为 Discovery、TidbMonitor 和 TidbInitializer 设置自定义 `labels` 与 `annotations` ([#4029](https://github.com/pingcap/tidb-operator/pull/4029), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (qiancai) [inline zh/releases/release-1.2.0-rc.2.md:15]:
```suggestion
- 支持为 Discovery、TidbMonitor 和 TidbInitializer 设置自定义的 `labels` 与 `annotations` ([#4029](https://github.com/pingcap/tidb-operator/pull/4029), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: zh/releases/release-1.2.0-rc.2.md (hunk 2 above)
Comment (csuzhangxc) [reply zh/releases/release-1.2.0-rc.2.md:22]: 有中文文档是用 `探测器`，是 K8s 里的一个术语。好处就是能方便识别是否 live/ready

Comment (qiancai) [reply zh/releases/release-1.2.0-rc.2.md:22]:
```suggestion
- 为 TiDB Operator 增加 liveness 与 readiness 探测器，便于查看 TiDB Operator 状态 ([#4002](https://github.com/pingcap/tidb-operator/pull/4002), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.2.0-rc.2.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+- Support setting `StorageVolumes`, `AdditionalVolumes`, and `AdditionalVolumeMounts` for TiCDC ([#4004](https://github.com/pingcap/tidb-operator/pull/4004), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support setting custom `labels` and `annotations` for Discovery, TidbMonitor, and TidbInitializer ([#4029](https://github.com/pingcap/tidb-operator/pull/4029), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support modifying Grafana dashboard ([#4035](https://github.com/pingcap/tidb-operator/pull/4035), [@mikechengwei](https://github.com/mikechengwei))
+
+## Improvements
+
+- Support using the TiKV version as the tag for BR `toolImage` if no tag is specified ([#4048](https://github.com/pingcap/tidb-operator/pull/4048), [@KanShiori](https://github.com/KanShiori))
+- Support handling PVC during scaling of TiDB ([#4033](https://github.com/pingcap/tidb-operator/pull/4033), [@csuzhangxc](https://github.com/csuzhangxc))
+- Add liveness and readiness probe for TiDB Operator ([#4002](https://github.com/pingcap/tidb-operator/pull/4002), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (qiancai) [inline en/releases/release-1.2.0-rc.2.md:22]:
```suggestion
- Add liveness and readiness probe for TiDB Operator to check the TiDB Operator status ([#4002](https://github.com/pingcap/tidb-operator/pull/4002), [@mikechengwei](https://github.com/mikechengwei))
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**